src/main/java/LoginTest.java

```diff
@@ -30,11 +33,21 @@ void loginTest() throws InterruptedException {
         }
 
         open(basicURL);
-        $(By.name("loginButton")).shouldBe(Condition.visible);
+        loginPage
+                .getLoginButton()
+                .shouldBe(Condition.visible);
```

Comment:
very minor, but I suggest using static import for Condition. code will be prettier